### PR TITLE
:hammer: Added changes to checker.py

### PR DESCRIPTION
**Pull request by yipkhenlai**
**Description**
To prevent errors running script happened when the folder is already created.

### CODE REVIEW

File: scripts/checker.py [hunk 1]
```diff
@@ -13,7 +13,11 @@
     targeted_path = os.getcwd()    # user also can modify the path they want save their unsupported images, exp : change the os.getcwd() to "C:\\Users\\ken\\Documents"
     directory = "UnsupportedImg"
     path = os.path.join(targeted_path, directory)
-    os.mkdir(path)
+
+    if path == None:
+        os.mkdir(path)
+    else:
```
Comment (codenamewei) [inline scripts/checker.py]: can remove the else loop

**Review by codenamewei — CHANGES_REQUESTED** [review]

File: scripts/checker.py (hunk 1 above)
Comment (yipkhenlai) [reply scripts/checker.py]: @codenamewei ok

**Review by codenamewei — CHANGES_REQUESTED** [review]
I don't seems to able to enter the if loop with your code
when the path do not exist

Screenshot 2021-06-18 at 9 26 16 AM

**Comment by yipkhenlai** [comment]
@codenamewei You are right, I will correct the mistake.

**Review by codenamewei — APPROVED** [review]

**Pull request merged**